src/Payment_link.php

```diff
@@ -16,6 +16,8 @@ class Payment_link extends Entity
      */
     public function create($attributes = array())
     {
+        $attributes = json_encode($attributes);
```

Comment:
@abhayp @neera11 , why we need this conversion when other API on RZP support for both json and form data as well.

Reply:
@abhayp Can you please clarify.As discussed earlier it is constraint due to new APIs written in go lang.